vscode-dotnet-runtime-library/src/Acquisition/IDotnetResolver.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+*  Licensed to the .NET Foundation under one or more agreements.
+*  The .NET Foundation licenses this file to you under the MIT license.
+*--------------------------------------------------------------------------------------------*/
+
+import { DotnetInstallMode } from './DotnetInstallMode';
+import { IDotnetListInfo } from './IDotnetListInfo';
+
+export interface IDotnetResolver
+{
+    getDotnetInstalls(dotnetExecutablePath: string, mode: DotnetInstallMode, requestedArchitecture: string | undefined | null): Promise<IDotnetListInfo[]>
```

Comment:
Should jsdoc comments be added here to clarify what these methods do?

Reply:
Great idea, added. Thanks 😄